osu.Game/Screens/Play/HUD/HoldForMenuButton.cs

```diff
@@ -60,6 +60,18 @@ public HoldForMenuButton()
             AutoSizeAxes = Axes.Both;
         }
 
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            var activationDelay = config.GetBindable<int>(OsuSetting.UIHoldActivationDelay).GetBoundCopy();
+            activationDelay.BindValueChanged(v =>
```

Comment:
Generally, we should bind value changed events in `load()`. Move to `LoadComplete()

Reply:
Adding why would be much helpful to avoid that.

Reply:
load is potentially in an async context. if the bound bindable changes during load, it may trigger a `ValueChanged` which isn't prepared for this to happen (bindables are generally considered update-thread-only).

You can fix this by adding a `Schedule` in the value changed event, but as a rule of thumb we avoid this practice to keep things simple.